### PR TITLE
Add workflow id to task table

### PR DESCRIPTION
**Pull request by Beacze**
Needed to track changes done to tasks.

Depends on #1420

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java [hunk 1]
```diff
@@ -43,6 +44,7 @@
      *            * determined out of sequence flow
      */
     public KitodoTask(Task task, int ordering) {
+        this.workflowId = task.getId();
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java]: You are sure that this assignment is correct? Should it not be `task.getWorkflowId()`?

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java]: Yes, it is correct. Task object is a task from BPMN model (not our bean).

Comment (henning-gerhardt) [reply Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java]: This is a little bit confusing....

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java]: I know but I can't change name of the class inside the bpmn model library.

**Pull request merged**